include/rocksdb/wide_columns.h

```diff
@@ -232,6 +232,8 @@ class PinnableAttributeGroup {
   explicit PinnableAttributeGroup(ColumnFamilyHandle* column_family)
       : column_family_(column_family), status_(Status::OK()) {}
 
+  explicit PinnableAttributeGroup(const Status& status) : status_(status) {}
+
```

Comment:
I think if we treat empty `PinnableAttriuteGroups` objects as no-op, this won't be necessary, right?

Reply:
Correct